lms/djangoapps/api_manager/courses/tests.py

```diff
@@ -2148,6 +2148,19 @@ def test_courses_data_metrics(self):
         response = self.do_get(course_metrics_uri)
         self.assertEqual(response.status_code, 404)
 
+    def test_course_data_metrics_user_group_filter(self):
```

Comment:
This test only check one scenario where are filtering user with none existing groups. I think we should more tests to cover different scenarios like users in different groups, mix of users in some group but not in others.